src/openms/source/ANALYSIS/OPENSWATH/MRMFeatureFinderScoring.cpp

```diff
@@ -150,8 +150,10 @@ namespace OpenMS
     scores_to_use.setValidStrings("use_ms1_fullscan", ListUtils::create<String>("true,false"));
     scores_to_use.setValue("use_ms1_mi", "false", "Use the MS1 MI score", ListUtils::create<String>("advanced"));
     scores_to_use.setValidStrings("use_ms1_mi", ListUtils::create<String>("true,false"));
-    scores_to_use.setValue("use_uis_scores", "false", "Use UIS scores for peptidoform identification ", ListUtils::create<String>("advanced"));
+    scores_to_use.setValue("use_uis_scores", "false", "Use UIS scores for peptidoform identification", ListUtils::create<String>("advanced"));
     scores_to_use.setValidStrings("use_uis_scores", ListUtils::create<String>("true,false"));
+    scores_to_use.setValue("use_ionseries_scores", "true", "Use MS2-level b/y ion-series scores for peptidoform identification", ListUtils::create<String>("advanced"));
```

Comment:
Just for my understanding, this score was "just" decoupled from the DIAScores, or?

Reply:
Basically a sub-score. If DIAScores is off, I want this to be off, too. But I also want to be able to disable it, even if DIAScores are "on".